node/src/chain_spec.rs

```diff
@@ -108,20 +113,44 @@ pub fn local_testnet_config() -> Result<ChainSpec, String> {
 
 /// Configure initial storage state for FRAME modules.
 fn testnet_genesis(
-    initial_authorities: Vec<(SpinId, GrandpaId)>,
+    initial_authorities: Vec<(AccountId, AccountId, SpinId, GrandpaId)>,
     root_key: AccountId,
     endowed_accounts: Vec<AccountId>,
     _enable_println: bool,
 ) -> serde_json::Value {
+    // Configure endowed accounts with initial balance of 1 << 63.
+    const ENDOWMENT: u128 = 10u128.pow(6) * qf_runtime::UNIT;
+    const STASH: u64 = 1u64 << 31;
```

Comment:
Less then the existential deposit. Maybe set equal to `ENDOWMENT` or something like `qf_runtime::EXISTENTIAL_DEPOSIT + qf_runtime::UNIT`?

Reply:
Set 10^5 UNIT for now